test/bigtable/index.js

```diff
@@ -0,0 +1,313 @@
+  this.calledWith_ = arguments;
+  Table.apply(this, arguments);
+}
+
+describe('Bigtable', function() {
+  var PROJECT_ID = 'test-project';
+  var ZONE = 'test-zone';
+  var CLUSTER = 'test-cluster';
+
+  var CLUSTER_NAME = format('projects/{p}/zones/{z}/clusters/{c}', {
+    p: PROJECT_ID,
+    z: ZONE,
+    c: CLUSTER
+  });
+
+  var Bigtable;
+  var bigtable;
+
+  before(function() {
+    mockery.registerMock('../../lib/common/grpc-service.js', FakeGrpcService);
+    mockery.registerMock('../../lib/common/util.js', fakeUtil);
+    mockery.registerMock('../../lib/bigtable/table.js', FakeTable);
+
+    mockery.enable({
+      useCleanCache: true,
+      warnOnUnregistered: false
+    });
+
+    Bigtable = require('../../lib/bigtable/index.js');
+  });
+
+  after(function() {
+    mockery.deregisterAll();
+    mockery.disable();
+  });
+
+  beforeEach(function() {
+    bigtable = new Bigtable({
+      projectId: PROJECT_ID,
+      zone: ZONE,
+      cluster: CLUSTER
+    });
+  });
+
+  describe('instantiation', function() {
+    it('should normalize the arguments', function() {
+      var normalizeArguments = fakeUtil.normalizeArguments;
+      var normalizeArgumentsCalled = false;
+      var fakeOptions = {
+        projectId: PROJECT_ID,
+        zone: ZONE,
+        cluster: CLUSTER
+      };
+      var fakeContext = {};
+
+      fakeUtil.normalizeArguments = function(context, options) {
+        normalizeArgumentsCalled = true;
+        assert.strictEqual(context, fakeContext);
+        assert.strictEqual(options, fakeOptions);
+        return options;
+      };
+
+      Bigtable.call(fakeContext, fakeOptions);
+      assert(normalizeArgumentsCalled);
+
+      fakeUtil.normalizeArguments = normalizeArguments;
+    });
+
+    it('should localize the cluster name', function() {
+      assert(bigtable.clusterName, CLUSTER_NAME);
+    });
+
+    it('should inherit from GrpcService', function() {
+      assert(bigtable instanceof GrpcService);
+
+      var calledWith = bigtable.calledWith_[0];
+
+      assert.strictEqual(calledWith.baseUrl, 'bigtable.googleapis.com');
+      assert.strictEqual(calledWith.service, 'bigtable');
+      assert.strictEqual(calledWith.apiVersion, 'v1');
+
+      assert.deepEqual(calledWith.protoServices, {
+        BigtableService: googleProtoFiles.bigtable.v1,
+        BigtableTableService: {
+          path: googleProtoFiles.bigtable.admin,
+          service: 'bigtable.admin.table'
+        }
+      });
+
+      assert.deepEqual(calledWith.scopes, [
+        'https://www.googleapis.com/auth/bigtable.admin',
+        'https://www.googleapis.com/auth/bigtable.data'
+      ]);
+    });
+  });
+
+  describe('formatTableName_', function() {
+    it('should return the last section of a formatted table name', function() {
+      var fakeTableName = 'projects/p/zones/z/clusters/c/tables/my-table';
+      var formatted = Bigtable.formatTableName_(fakeTableName);
+
+      assert(formatted, 'my-table');
```

Comment:
I believe this is just checking that `formatted` is a truthy value. We want `assert.strictEqual`.